wiki/History_of_osu!/2016/fr.md

```diff
@@ -1,132 +1,182 @@
-- [osu!weekly \#49](https://osu.ppy.sh/home/news/2016-02-23-osuweekly-49)
+De plus, en réponse à la réaction de la communauté, Ephemeral a promis que les changements apportés aux critères de classement seraient temporairement suspendus et que l'équipe d'osu! s'engagerait à être plus transparente dans ses processus :
+
+> À l'avenir, nous visons à être entièrement transparents avec nos processus de réflexion menant aux grandes décisions qui ont un impact sur toute la communauté.
+>
+> ...
+>
+> Nous espérons qu'en plaçant les événements qui ont conduit à ces décisions à la vue du public, nous pourrons travailler ensemble pour trouver la meilleure façon d'avancer à partir d'ici et pour nous assurer qu'une telle chose ne se reproduise plus.
+>
+> Nous avons suspendu tout mouvement dans les changements des critères de classement pour (au moins) le reste de la semaine, et nous utiliserons ce temps pour rassembler vos avis et nos réflexions sur la meilleure façon de procéder à partir de maintenant.
+
+—Ephemeral, "Meet Ephemeral!" [[4,5]][r]
+
+## Mars
+
+**Le premier jour du mois de mars, Ephemeral a créé le premier post d'une série sur le blog de peppy qui deviendrait les "Meeting Notes", qui ont servi à tenter de fournir plus de transparence entre l'équipe d'osu! et la communauté.** Dans ces posts, Ephemeral résumerait les pensées, idées, et objectifs principaux que l'équipe aurait dans le futur en plusieurs points. Ce premier post visait principalement à améliorer l'équipe et sa communication à tous les niveaux (en mettant l'accent sur le beatmapping et le modding) [[8]][r].
 
 ## Avril
 
-La prise en charge de l'Oculus Rift en tant que dispositif d'entrée a été annoncée en 2016 en tant que Poisson d'Avril.
+**La panel de modding du nouveau site d'osu! (new.ppy.sh) a été mis en ligne afin de permettre des tests publics le 11 avril 2016.** Cependant, ces tests n'étaient disponibles que pour [Ice - L2 - Ascension : Act 2 (Liberation) (LordRaika)](https://osu.ppy.sh/beatmapsets/123760) et [Camellia - overcomplexification (ProfessionalBox)](https://osu.ppy.sh/beatmapsets/351408). Selon [Loctav](https://osu.ppy.sh/users/71366), l'idée principale du panel était de diviser les problèmes en posts séparés, puis d'en discuter individuellement pour obtenir un suivi plus facile des problèmes et de s'en servir comme une partie des étapes vers la mise en œuvre du "Modding V2" tant attendu [[9]][r].
 
-[Maria](/wiki/Mascots#mani-&-mari) a rejoint la scène avec [pippi](/wiki/Mascots#pippi) et [Yuzu](/wiki/Mascots#yuzu) en tant que nouvelle mascotte osu!mania. La osu!taiko World Cup 2016 s'est terminée avec une victoire du Japon face à Taïwan.
+**De nouveaux designs de médailles ont été ajoutés à osu! le 14 avril 2016.** Ces designs permettaient d'être plus en phase avec le design d'"[osu!next](https://osunext.tumblr.com/about)" qui était utilisé dans le nouveau client d'osu! (nom de code "lazer") et sur le nouveau site web [[10,11]][r].
```

Comment:
> d'"[osu!next](https://osunext.tumblr.com/about)"

this looks pretty weird

Reply:
yes, I originally used `d' "osu!next"` but it would still look weird imo, I'll rephrase that